src/raw/client.rs

```diff
@@ -279,6 +279,17 @@ impl Client {
         ResponseType::from_response(response)
     }
 
+    /// Endpoint:
+    /// [`playlist`](https://listenbrainz.readthedocs.io/en/latest/users/api/playlist.html#get--1-playlist-(playlist_mbid))
+    /// (`GET`)
+    pub fn get_playlist(&self, playlist: &str) -> Result<GetPlaylistResponse, Error> {
+        let endpoint = format!("{}{}", self.api_root_url, Endpoint::Playlist(playlist));
```

Comment:
This can be simplified to `self.get(Endpoint::Playlist(playlist))` (I think)

Reply:
Oh, much nicer indeed, thanks!